Sources/SPMBuildCore/PluginInvocation.swift

```diff
@@ -321,6 +325,18 @@ public struct BuildToolPluginInvocationResult {
     /// The directory given to the plugin as a place in which it and the commands are allowed to write.
     public var pluginOutputDirectory: AbsolutePath
 
+    /// The package to which the plugin was applied.
+    public var package: ResolvedPackage
+
+    /// The target in that package to which the plugin was applied.
+    public var target: ResolvedTarget
+
+    /// If the plugin finished successfully.
+    public var succeeded: Bool
+
+    /// Duration in seconds of the plugin invocation.
+    public var duration: Double
```

Comment:
use Dispatch's TimeInterval?

Reply:
Good point, is that the de facto standard for this in SwiftPM?  In Foundation it was typically NSTimeInterval, which was defined as being a double in seconds.

Reply:
yes for now (there is a stdlib type coming)